### DISCUSSION TITLE
Ausweis der Gesamtleistung (kWh)- AC oder DC?

### DISCUSSION THREAD

**Discussion by 2000ede** [tbnobody/OpenDTU#2121] [Q&A]
Hallo,
sind die ausgewiesenen Werte

- "Gesamtertrag insgesamt" (Gesamtübersicht)
- "Gesamtertrag" (im Fenster Phase 1)
- "Gesamtertrag" (im Fenster String 1)

AC oder DC?

Ich tippe auf DC. - da alle Werte anscheinend gleich sind. 
Wenn das so ist, wird AC gar nicht ausgewiesen wird?
Warum nicht, bei der aktuellen Leistung wird ja auch der Wirkungsgrad mit ermittelt.

Danke für Eure Hinweise!

**Comment by Juergen2453** [comment]
**Anzeige nur ein Inverter, vielleicht hilft es.**

![x1](https://github.com/tbnobody/OpenDTU/assets/116970574/18603b90-bf90-4201-b8e8-383f40d8dbb0)

**Comment by 2000ede** [comment]
Hallo Jürgen,
meine Frage bezog sich nicht auf die Momentanwerte (Watt), sondern auf die "Zählerwerte" bzw kWh-Werte. In deiner Sicht u.a. oben links.

Die Momentanwerte sind mir klar.

Dies scheinen alles DC Werte zu sein:
![grafik](https://github.com/tbnobody/OpenDTU/assets/37264959/57dd0c82-4318-4338-8205-24f499079478)

**Comment by Juergen2453** [comment]
Die Anzeige ist verwirrend, Der Wert 1.875,287 kWh ist AC. Nimm die Differenz (Gesamtertrag Insgesamt) von 2 Tagen, das ich ja die "Gesamtertrag Heute" für einen Tag und das ist AC.

**Reply by 2000ede** [reply]
Nochmal: Ist die Gesamtleistung insgesamt wirklich AC?

In deinem Beispiel ergeben die beiden Module bzw. MPPTs genau die Gesamtleistung:
942.299
932.988
1.875.287

Würde ja bedeuten, dass die Werte bei den Modulen auch AC wären. Das glaube ich nicht.
Zumal im json "Inverters Dc 0 Yieldtotal" steht.

Für mich sind das damit alles DC Werte. Der relevante AC Wert fehlt - oder ich mache immer noch einen Denkfehler.

**Comment by Juergen2453** [comment]
MQTT Explorer. Die Werte haben sich natürlich minimal geändert bei "Yield Total".


![x2](https://github.com/tbnobody/OpenDTU/assets/116970574/1b5b30ad-d707-4ee2-8e67-e2145942956a)

**Comment by 2000ede** [comment]
ich fürchte, wir kommen hier nicht weiter ohne einen Experten der die Anwendung genau kennt. 
@sfromis or @@MetaChuh : Könnt ihr helfen?

Wie gesagt, DC kwh (nicht w oder kw) muss höher sein als AC kwh. 
Und hier ergeben die Modulewerte (sollte DC sein) die Gesamtzahl. Dann kann das m.E. nicht AC sein.

Daher nochmal zusammenfassend:
- Welcher Wert ist DC und welcher AC von den kwh-Werten?
- Wo finde ich die jeweiligen "Gegenwerte" in DC bzw AC? Besonders der "Zählerwert / Gesamtwert in AC" oder wenn er in AC ist in DC?

![grafik](https://github.com/tbnobody/OpenDTU/assets/37264959/c02acef3-ce3b-494d-a32f-086afc3ae61f)
